plugin/src/lib.rs

```diff
@@ -724,6 +724,7 @@ fn fold_first_block(block: P<Block>, m: &mut MutatorPlugin) -> P<Block> {
                         .unwrap(),
                 );
             }
+            //let ($a, $b) = if mutagen::now($n) { ($b, $a) } else { ($a, $b) };
             //TODO: switch interchangeables, need mutability info, too
             //for name in method_info.interchangeables { }
```

Comment:
You should uncomment this loop. The `name' is wrong, we need `(ref key, ref values)` pairs, then iterate over `values`. Now in that inner loop we have two `Symbol`s we want to switch (to use them wihin a `quote!(..)`, we need to get their Identifier, so call `.to_ident()` as above in line 713.

Next, we need to add the mutation to our mutation list (so add_mutations(..) with a single mutation. The text should be `"switch the {key} and {value} argument"` (format! with key and value). See lines 712 and following. Finally, we need to push our mutation statement, which we can get by `quote!`ing the code in line 727's comment (just change `$a` and `$b` as necessary).